paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -244,7 +244,7 @@ internal class CustomerSheetViewModel @Inject constructor(
                         enabled = true,
                         isProcessing = false,
                         primaryButtonEnabled = it.formViewData.completeFormValues != null,
-                        errorMessage = result.throwable.stripeErrorMessage(application),
+                        errorMessage = stripeErrorMessageProvider(result.throwable),
```

Comment:
We could also think about making `stripeErrorMessage()` return a `ResolvableString`, but this is fine for now.